pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -714,6 +714,9 @@ type GithubEventSource struct {
 	// Repositories holds the information of repositories, which uses repo owner as the key,
 	// and list of repo names as the value
 	Repositories []OwnedRepositories `json:"repositories,omitempty" protobuf:"bytes,15,rep,name=repositories"`
+	// GitHub Insecure TLS verification (for GitHub Enterprise)
+	// +optional
+	GithubInsecure bool `json:"githubInsecure,omitempty" protobuf:"varint,16,opt,name=githubInsecure"`
```

Comment:
Maybe `SkipVerifyTLS`? And give comments like below:

Whether to skip TLS verification when connecting to Github for webhook creation, useful for Github Enterprise use case.